<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ppt_builder</class>
 <widget class="QMainWindow" name="ppt_builder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PPT Builder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>lamp.ico</normaloff>lamp.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(225, 225, 225);
}
QCheckBox{
width:130%;
}
QCheckBox:hover{
color:rgb(100,100,100);
}
QTextEdit{
border: 2px solid rgb(238, 238, 238);
border-radius: 6px;
padding: 5px;
background-color: white;
}
QTextBrowser{
border: 2px solid rgb(238, 238, 238);
border-radius: 6px;
padding: 5px;
background-color: white;
}
QLineEdit{
border: 2px solid rgb(238, 238, 238);
border-radius: 6px;
background-color: white;
}
QLabel{
padding-top:5px;
color: black;
font: bold;
}
QPushButton {
color: black;
border: 2px solid rgb(204, 204, 204);
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.2, cy: 1.3,
fx: 1.0, fy: 0.5,
radius: 1.35, stop: 0 rgb(228, 228, 228), stop: 1 rgb(208, 208, 208));
min-width: 80px;
font: bold;
}
QPushButton:pressed {
color:blue;
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 rgb(245, 245, 245));
font:italic;
}
QPushButton:hover{
color:rgb(100,100,100);
}
QScrollBar::horizontal {
max-height: 7px;
}
QScrollBar::vertical {
max-width: 7px;
}
QScrollBar::handle:vertical {
background-color:rgb(200, 200, 200);
max-width: 4px;
}
QScrollBar::handle:horizontal {
background-color:rgb(200, 200, 200);
max-height: 4px;
}</string>
  </property>
  <widget class="QWidget" name="MainWindow">
   <property name="styleSheet">
    <string notr="true">opacity: 0;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="text_output">
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="statusTip">
         <string>Progress Bar</string>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>35</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="statusTip">
           <string>Click to start</string>
          </property>
          <property name="text">
           <string>Build Slides</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="slide_to_use">
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Slide number from template's master view (Changing may results in strange slides or errors!)
Change this only in the event of a new ppt template</string>
          </property>
          <property name="statusTip">
           <string>Slide number from template's master view (Changing may results in strange slides or errors!)</string>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Leave</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionView_pharma_avoids">
   <property name="text">
    <string>View Pharma Avoids</string>
   </property>
   <property name="statusTip">
    <string>View Pharma Avoids</string>
   </property>
  </action>
  <action name="actionView_inn_avoids">
   <property name="text">
    <string>View INN Avoids</string>
   </property>
  </action>
  <action name="actionProject_avoids">
   <property name="text">
    <string>Project Avoids</string>
   </property>
  </action>
  <action name="actionInternal_names">
   <property name="text">
    <string>Internal Names</string>
   </property>
  </action>
  <action name="actionPresented_names">
   <property name="text">
    <string>Presented Names</string>
   </property>
  </action>
  <action name="actionCompetitor_names">
   <property name="text">
    <string>Competitor Names</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="statusTip">
    <string>View Options</string>
   </property>
  </action>
  <action name="actionTool_1">
   <property name="text">
    <string>Tool 1</string>
   </property>
  </action>
  <action name="actionTool_style">
   <property name="text">
    <string>Tool 2</string>
   </property>
  </action>
  <action name="actionTool_3">
   <property name="text">
    <string>Tool 3</string>
   </property>
  </action>
  <action name="actionTool_4">
   <property name="text">
    <string>Tool 4</string>
   </property>
  </action>
  <action name="actionTool_5">
   <property name="text">
    <string>Tool 5</string>
   </property>
  </action>
  <action name="actionSave_names">
   <property name="text">
    <string>Commit Names</string>
   </property>
  </action>
  <action name="actionSort_names">
   <property name="text">
    <string>Sort Names (A-Z)</string>
   </property>
  </action>
  <action name="actionFind_conflicts">
   <property name="text">
    <string>Find Conflicts</string>
   </property>
  </action>
  <action name="actionClear_avoids">
   <property name="text">
    <string>Clear All Avoids</string>
   </property>
  </action>
  <action name="actionView_stripped_names">
   <property name="text">
    <string>View names without rationale</string>
   </property>
  </action>
  <action name="actionINN_avoids_password_required">
   <property name="text">
    <string>INN avoids (password req.)</string>
   </property>
  </action>
  <action name="actionPharma_Avoids_password_req">
   <property name="text">
    <string>Pharma Avoids (password req.)</string>
   </property>
  </action>
  <action name="actionINN_avoids">
   <property name="text">
    <string>INN Avoids</string>
   </property>
  </action>
  <action name="actionPharma_avoids">
   <property name="text">
    <string>Pharma Avoids</string>
   </property>
  </action>
  <action name="actionAdd_avoids">
   <property name="text">
    <string>Add Avoids</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
